tests/test_specs/reserve-transfer-asset-xcm.spec.ts

```diff
@@ -147,3 +147,129 @@ test.describe('Test case: XCM001', () => {
     expect(difference).toEqual(transferAmount);
   });
 });
+
+test.describe('Test case: XCM001-1', () => {
+  test('should transfer Alice DOT tokens from Polkadot to Astar', async ({ page, context }) => {
+    const assetId = '340282366920938463463374607431768211455';
```

Comment:
Where the `assetId` variable has been used in this test function?

Reply:
in the `getBalance(ALICE_ADDRESS, assetId) ` functions, this allows to check balances other than just ASTR.